src/core/public/styles/rendering/_base.scss

```diff
@@ -63,15 +56,24 @@
   }
 }
 
+// Set a body CSS variable for the app container to use - calculates the total
+// height of all fixed headers + the sticky action menu toolbar
+.kbnBody--projectActionMenuVisible {
```

Comment:
I pulled this out to its own selector block instead of bogarting the other existing selectors because I thought it was easier to read and understand separately. If the toolbar goes away or changes its positioning logic someday, separating it also makes it easier to delete all at once